pkg/nodebootstrap/assets/10-eksclt.al2.conf

```diff
@@ -5,15 +5,15 @@
 EnvironmentFile=/etc/eksctl/metadata.env
 # Global and static parameters: MAX_PODS, CLUSTER_DNS, NODE_LABELS
 EnvironmentFile=/etc/eksctl/kubelet.env
-# Local non-static parameters: NODE_IP
+# Local non-static parameters: NODE_IP, INSTANCE_ID
 EnvironmentFile=/etc/eksctl/kubelet.local.env
 
 ExecStart=
 ExecStart=/usr/bin/kubelet \
   --node-ip=${NODE_IP} \
   --cluster-dns=${CLUSTER_DNS} \
   --max-pods=${MAX_PODS} \
-  --node-labels=${NODE_LABELS} \
+  --node-labels=${NODE_LABELS},alpha.eksctl.io/instance-id=${INSTANCE_ID} \
```

Comment:
How does this work when `NODE_LABELS` is empty?

Reply:
`kubelet` just ignores it.

Also, practically speaking, it never becomes empty because of the cluster-name and the node-group-name labels.

A typical `kubelet.env` that fills the NODE_LABELS env looks like this:

```
NODE_LABELS=alpha.eksctl.io/nodegroup-name=nodegroup1,alpha.eksctl.io/cluster-name=mycluster
```

Reply:
You are right, it used to be empty until recently, and now we always set those labels. Thanks!

Reply:
Thanks for confirming and providing the context!